pyproject.toml

```diff
@@ -202,5 +190,5 @@ lint = [{task="ruff"}, {task="pre-commit"}]
 ci = [{task="lint"}, {task="unit"}, {task="integration"}, {task="coverage"}]
 
 [tool.pixi.dependencies]
-gdal = "==3.11.0"
+gdal = ">=3.11.0,<4"
```

Comment:
Is this going to cause incompatibilities with the GDAL in PUDL when there's a minor update?